cli/src/cli_util.rs

```diff
@@ -2420,10 +2433,12 @@ pub struct GlobalArgs {
     /// Operation to load the repo at
     ///
     /// Operation to load the repo at. By default, Jujutsu loads the repo at the
-    /// most recent operation. You can use `--at-op=<operation ID>` to see what
-    /// the repo looked like at an earlier operation. For example `jj
-    /// --at-op=<operation ID> st` will show you what `jj st` would have
-    /// shown you when the given operation had just finished.
+    /// most recent operation. If there are concurrent operations, they will be
```

Comment:
(Optional) There is a bit of mixed tenses here: "loads ... they will be merged". One way to phrase this:

I'd reword this a bit:

> By default, Jujutsu loads the repo at the most recent operation, or at the merge of the latest concurrent operations if there is more than one.

I'd also consider adding a paragraph break.

Reply:
Thanks. Updated as
```
    /// Operation to load the repo at. By default, Jujutsu loads the repo at the
    /// most recent operation, or at the merge of the concurrent operations if
    /// any.
    ///
    /// You can use `--at-op=<operation ID>` to see what the repo looked like at
```